// Copyright 2020 The Monyhar Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

package org.monyhar.chrome.browser;

import org.monyhar.chrome.browser.base.SplitCompatBackupAgent;
import org.monyhar.chrome.browser.base.SplitCompatUtils;

/** See {@link ChromeBackupAgentImpl}. */
public class ChromeBackupAgent extends SplitCompatBackupAgent {
    public ChromeBackupAgent() {
        super(SplitCompatUtils.getIdentifierName(
                "org.monyhar.chrome.browser.ChromeBackupAgentImpl"));
    }
}
